scripts/setup.sh

```diff
@@ -44,6 +44,9 @@ function run() {
     elif [[ -d ${GLOBAL_CLANG} ]]; then
       echo "Detected '${GLOBAL_CLANG}/bin/clang++', using it"
       local llvm_toolchain_dir=${GLOBAL_CLANG}
+    elif [[ "${MASON_LLVM_RELEASE}" == "system" ]]; then
```

Comment:
This was the only thing I was missing all along, I even tried to upgrade llvm in mason packages. Thanks for fixing this. 😄